console/src/components/Services/Data/TableRelationships/RemoteRelationships/components/RemoteRelEditor.tsx

```diff
@@ -0,0 +1,158 @@
+import React from 'react';
+import {
+  RemoteRelationship,
+  TreeArgElement,
+  TreeFieldElement,
+  ArgValueKind,
+} from '../utils';
+import {
+  Action as RemoteRelAction,
+  setName,
+  setRemoteSchema,
+  setArgValue,
+  setArgValueKind,
+  toggleArg,
+  toggleField,
+} from '../state';
+
+import styles from '../SchemaExplorer.scss';
+import {
+  RelName as RelNameTooltip,
+  RemoteSchema as RemoteSchemaTooltip,
+  Configuration as ConfigTooltip,
+} from '../Tooltips';
+import Explorer from './Explorer';
+
+type Props = {
+  table: any; // use "Table" type after ST is merged
```

Comment:
```suggestion
  table: any; // TODO: use "Table" type after ST is merged
```
So we can grep by "TODO"